### ISSUE TITLE
Add an option to open groups in a new window as default

### ISSUE THREAD

**Issue by ETx13** [Drive4ik/simple-tab-groups#1133]
I know you can right click and open in a new window, but I feel like it would be nice to have an option to set that as a default.

**Comment by krishhandro** [comment]
I agree. I have a few specific groups and I always open them in new windows. It would be nice to avoid the extra step to right-click on the group names.

**Comment by Liquidream** [comment]
Exactly what I'm looking for also - really hope the option gets added.

Having used Edge prior to Firefox for quite some time (+still needing to for work), this extension is giving me the closest functionality to Grouped tabs + Workspaces in Firefox that I've seen yet. Alas, the default (in Edge) that I'm used to is that when you select a Workspace from the drop-down - it'll open in new window by default (or switch to window if already open).

So yeah, having the ability to change the default to this in settings (so that we don't have to right-click each time) would be awesome 🙂

**Comment by ncflagg** [comment]
+1

Also, is it normal that opening a group deletes all tabs in the current Firefox window? I lost a few windows before I realized I MUST either:
  -Right-click and 'Open group in new window'
      OR
  -Ctrl+N to open a new blank window and then open a group

Otherwise, like I said, the default behavior destroys all open tabs. They don't even appear in the Firefox History. I think they end up in 'Show other tabs' (hidden) in STG. Seems crazy to me that this would be the default behavior
